__tests__/Hospital-Detail /ContactCard.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ContactCard from "./../../components/hospital-detail/ContactCard";
+
+const mockData = {
```

Comment:
It is ok to leave this mock data here since, if you are going to use it only here.